docs/SERVER.md

```diff
@@ -29,6 +29,7 @@ Use curl or your favorite tool to pass arguments to the `/sbom` route.
 
 View the latest [cdxgen server API specification](https://github.com/CycloneDX/cdxgen/blob/master/lib/server/openapi.yaml) online via [Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/CycloneDX/cdxgen/master/lib/server/openapi.yaml).
 
+[API Docs](./api-docs.html ':include :type=iframe')
```

Comment:
Let's avoid html. Since this load an html with the remote `cdn.redoc.ly`, a supply chain attack on their cdn, will affect our users. A while ago, I faced a similar issue due to an attack on video.js cdn.

Reply:
Closing then. I'll raise issues before doing work in future

Reply:
Apologies mate.